### PR TITLE
Find single characters rather than single character strings.

### PR DESCRIPTION
**Pull request by Dave-Lowndes**
Remove unused variable.

### CODE REVIEW

File: include/TuningsImpl.h [hunk 1]
```diff
@@ -29,7 +29,6 @@ inline std::istream &getlineEndingIndependent(std::istream &is, std::string &t)
 {
     t.clear();
```
Comment (baconpaul) [inline include/TuningsImpl.h:31]: Ooooh removing the sentry is not correct. Thats a guard class which sets internal bits

Comment (Dave-Lowndes) [reply include/TuningsImpl.h:31]: @baconpaul 
OK, so perhaps the fix should be to test the sentry object is valid after construction?
That way it wouldn't be flagged as unused by an analyser.

Comment (Dave-Lowndes) [reply include/TuningsImpl.h:31]: @baconpaul See what you think of the latest commit.

**Comment by baconpaul** [comment]
Great!

**Pull request merged**